torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -218,7 +300,7 @@ std::vector<Node*> getOnnxConstParentsToRemove(Node* node) {
 
 // This method updates the block in-place to fold all the one-time
 // constant-based computations/ops into an initializer node.
-void ConstantFoldONNX(Block* b, ParamMap& paramsDict) {
+void ConstantFoldONNX(Block* b, ParamMap& paramsDict, int opset_version) {
   AT_ASSERT(b->param_node());
```

Comment:
We may also want to add a check on `opset_version` here as well. If it's not supported, skipping should be enough.

Reply:
Good point. Done.